includes/controllers/pages/class-submit-listing.php

```diff
@@ -1061,7 +1061,7 @@ private function plan_selection() {
 
 		// Returns null if value isn't posted.
 		$categories      = $category_field->value_from_POST();
-		$should_validate = ! empty( $_POST ) && ( ! empty( $categories ) || $categories !== null );
+		$should_validate = ! empty( $_POST ) && ! empty( $_POST ['current_section'] ) && $_POST ['current_section'] === 'plan_selection';
```

Comment:
Let's use the same function we used in other PRs to get the sanitized POST value.